contrib/test_jenkins.sh

```diff
@@ -832,12 +832,28 @@ test_ucs_load() {
 	! grep '^socket' strace.log
 }
 
-test_dlopen() {
+test_ucs_dlopen() {
 	$MAKEP
 
 	# Make sure UCM is not unloaded
-	echo "==== Running dlopen test with memhooks ===="
-	./test/apps/test_dlopen
+	echo "==== Running UCS dlopen test with memhooks ===="
+	./test/apps/test_ucs_dlopen
+}
+
+test_ucp_dlopen() {
+	../contrib/configure-release --prefix=$ucx_inst
+	$MAKEP clean
+	$MAKEP
+
+	# Make sure UCP library, when opened with dlopen(), loads CMA module
+	if find -name libuct_cma.so.0
+	then
+		echo "==== Running UCP library loading test ===="
+		./test/apps/test_ucp_dlopen # just to save output to log
+		./test/apps/test_ucp_dlopen | grep 'cma/cma'
```

Comment:
what is the aim of this grep?

Reply:
it should check that cma is loaded, by check that UCP has cma dev on cma transport (hence cma/cma)